<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>showtimer</class>
 <widget class="QWidget" name="showtimer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1326</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Show Timer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Uke nr :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_ukenr">
       <property name="correctionMode">
        <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
       </property>
       <property name="stepType">
        <enum>QAbstractSpinBox::DefaultStepType</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>SQLite sp√∏rring :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_sqlquery"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_save">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>dato</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>kunde</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ordrenr</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>category</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>rate_card</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>overtid</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>timer</string>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>timepris</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>notes</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>registrert</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>overtidlevert</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>uke</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
